src/components/common/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,44 @@
+import { AvatarShapeType } from '@/types/user';
+import { getAvatarSize } from '@/utils/getAvatarSize';
+
+import { AvatarContainerStyled, EditButtonStyled, ProfileImageStyled } from './Avatar.style';
+
+interface AvatarProps {
+  avatarShape: AvatarShapeType['avatarShape'];
+  profileImageSrc?: string;
+  isEditable?: boolean;
+  onClick?: () => void;
+}
```

Comment:
@colorkite10 p1;
채연님도 onClick에 대한 타입 정의때문에 고민 많이 하셨을 것 같아요!
```suggestion
  interface AvatarProps extends React.HTMLAttributes<HTMLDivElement> {
  avatarShape: AvatarShapeType['avatarShape'];
  profileImageSrc?: string;
  isEditable?: boolean;
}
```
이렇게 하면 onClick의 타입을 HTMLDivElement에서 상속받아 사용할 수 있습니다!

1.HTMLDivElement 설명 https://developer.mozilla.org/ko/docs/Web/API/HTMLDivElement
2.HTMLDIvElement에 선언된 타입들
https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.htmldivelement.html#onclick

Reply:
오옹~ 저도 활용해야겠군요! 꿀팁 감사합니다